src/stores/search.ts

```diff
@@ -32,6 +33,10 @@ import { useProviderStore } from '~/stores/provider'
 
 import { useFeatureFlagStore } from '~/stores/feature-flag'
 
+import { useStorage } from '~/composables/use-storage'
+
+import type { Ref } from '~/composables/types'
```

Comment:
`Ref` should be imported from the `@nuxtjs/composition-api' package.

```suggestion
import type { Ref } from '@nuxtjs/composition-api'
```